customer/java/microprofile/src/main/java/com/redhat/developer/demos/customer/rest/CustomerEndpoint.java

```diff
@@ -21,7 +21,7 @@
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
     @Inject
-    @ConfigProperty(name = "preferences.api.url", defaultValue = "http://preferences:8080")
+    @ConfigProperty(name = "preferences.api.url", defaultValue = "http://preference:8080")
```

Comment:
I'm not sure this is required for this PR, but the actual URL to be called, according to the trace information is `preference` (singular). So, I changed this default to match what's actually being used.